pkg/nvidia/cdi/jetson-xavier-nx.yaml

```diff
@@ -0,0 +1,3355 @@
+---
```

Comment:
Do you want perhaps add a comment on how you created these files?

Reply:
IMO would be better to have comments near the code, not far away. A link would be ok, too.

> that's why the post-processing is required to adjust pathnames of libraries, firmwares and other binary files

How is this done?

Reply:
I want to change the auto-generated yaml file as minimal as possible. The whole explanation is on the README.md file from the package, so I prefer to keep as it is....